test/NuGet.Core.Tests/NuGet.LibraryModel.Tests/FrameworkDependencyTests.cs

```diff
@@ -48,7 +48,7 @@ public void FrameworkDependecy_NamesWithDifferentCasign_DoNotAffectCompare()
         [Fact]
         public void FrameworkDependecy_DifferentFlags_AffectCompare()
         {
-            Assert.Equal(1, new FrameworkDependency("AAA", FrameworkDependencyFlags.All).CompareTo(new FrameworkDependency("AAA", FrameworkDependencyFlags.None)));
+            Assert.True(new FrameworkDependency("AAA", FrameworkDependencyFlags.All).CompareTo(new FrameworkDependency("AAA", FrameworkDependencyFlags.None)) > 0);
```

Comment:
I'm not sure I follow, what's the value that CompareTo returns?

Reply:
For net472 and netcoreapp2.1, it returns 1, 0 or -1. But according to the document, it should return relative values.
I found this test is broken when using a latest netcoreapp5.0 build. As netcoreapp5.0 returns the relative values.